packages/tools/locales/ru_RU.po

```diff
@@ -5091,377 +5086,3 @@ msgstr "Приблизить"
-
-#~ msgid "Tag name cannot start or end with a `/`."
-#~ msgstr "Название тега не может начинаться или оканчиваться на `/`."
-
-#~ msgid "Tag name cannot contain `//`."
-#~ msgstr "Название тега не может содержать `//`."
-
-#~ msgid "Add or remove tags"
-#~ msgstr "Добавить или удалить метки"
-
-#, fuzzy
-#~ msgid "Front"
-#~ msgstr "Перед"
-
-#~ msgid "Split"
-#~ msgstr "Раздельный вид"
-
-#~ msgid "Resources"
-#~ msgstr "Ресурсы: %d"
-
-#~ msgid "Global zoom percentage"
-#~ msgstr "Глобальный масштаб в процентах"
-
-#~ msgid "Writes all settings to STDOUT as JSON including secure variables."
-#~ msgstr ""
-#~ "Выводит в стандартный поток вывода все настройки в формате JSON (включая "
-#~ "конфиденциальные переменные)."
-
-#~ msgid "Reads in JSON formatted settings from STDIN."
-#~ msgstr "Считывает конфигурацию в формате JSON из потока стандартного ввода."
-
-#~ msgid "Reads in settings from <file>. <file> must contain valid JSON."
-#~ msgstr ""
-#~ "Загружает настройки из файла <file>. <file> должен содержать данные в "
-#~ "формате JSON."
-
-#~ msgid "Synchronisation is already in progress. State: %s"
-#~ msgstr "Синхронизация уже выполняется. Состояние: %s"
-
-#, fuzzy
-#~ msgid "Confirm master password:"
-#~ msgstr "Введите мастер-пароль:"
-
-#, fuzzy
-#~ msgid "Confirm password"
-#~ msgstr "Введите мастер-пароль:"
-
-#, fuzzy
-#~ msgid "Missing required argument: note"
-#~ msgstr "Отсутствует обязательный аргумент: %s"
-
-#~ msgid "Synchronisation status"
-#~ msgstr "Статус синхронизации"
-
-#~ msgid "General Options"
-#~ msgstr "Основные настройки"
-
-#~ msgid "Encryption options"
-#~ msgstr "Настройки шифрования"
-
-#~ msgid "Encryption Options"
-#~ msgstr "Настройки шифрования"
-
-#~ msgid "Clipper Options"
-#~ msgstr "Настройки клиппера"
-
-#, fuzzy
-#~ msgid "Permission to write to external storage"
-#~ msgstr "Разрешение на использование камеры"
-
-#~ msgid "Cancel synchronisation"
-#~ msgstr "Отменить синхронизацию"
-
-#~ msgid "Show metadata"
-#~ msgstr "Показать метаданные"
-
-#~ msgid "Delete notebook"
-#~ msgstr "Удалить блокнот"
-
-#~ msgid ""
-#~ "Click on the (+) button to create a new note or notebook. Click on the "
-#~ "side menu to access your existing notebooks."
-#~ msgstr ""
-#~ "Нажмите на кнопку (+) для создания новой заметки или нового блокнота. "
-#~ "Нажмите на боковое меню для доступа к вашим существующим блокнотам."
-
-#~ msgid ""
-#~ "You currently have no notebook. Create one by clicking on (+) button."
-#~ msgstr "У вас сейчас нет блокнота. Создайте его нажатием на кнопку (+)."
-
-#~ msgid "Welcome"
-#~ msgstr "Добро пожаловать"
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "Разделяйте каждую метку запятой."
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "Путь для синхронизации, когда включена синхронизация файловой системы. "
-#~ "См. `sync.target`."
```

Comment:
Please don't delete those. We keep old translations in the .po files.

Reply:
Okay. I just downloaded the .pot file and made a "rough translation" of the already translated .po

Reply:
What? I am very confused. Everything in the PR is great. Just don't delete the commented out strings. You only use the .pot file, if you start a new translation.

Reply:
Are you still working on this?